src/pages/Filing/FilingApp/FilingWarnings/FilingWarningsAlerts.tsx

```diff
@@ -31,18 +32,38 @@ function WarningsAlert(): JSX.Element {
   );
 }
 
+export function InstitutionFetchFailAlert({
+  isVisible = true,
+}: {
+  isVisible?: boolean;
+}): JSX.Element {
+  return (
+    <Alert
+      className='mb-[2.8125rem] [&_div]:max-w-[41.875rem] [&_p]:max-w-[41.875rem]'
+      message='The institution data service is unavailable'
+      status='error'
+      isVisible={isVisible}
+    >
+      There was a connection issue or our service may be temporarily
```

Comment:
This text appears in 3 places now I think? It might be worth making an error constants file, otherwise it's going to be weird to maintain it / make changes.

Reply:
@billhimmelsbach Did some consolidation here with a `AlertApiUnavailable` component that encompasses this text with a bit of configurability for the heading message and the link `href`.

Reply:
I think this makes sense @meissadia: we're going to have this on a ton of pages. I could see the error string just as a constant vs doing this full component, but I like where this is now and that it'll be easy to add all over the place. 👍